### DISCUSSION TITLE
Releasing new version of scoped-futures

### DISCUSSION THREAD

**Discussion by tlowerison** [weiznich/diesel_async#192] [General]
Hi @weiznich, I'm planning on releasing a new patch of [scoped-futures](https://github.com/tlowerison/scoped-futures) and wanted to give you a chance to review the changes since diesel-async currently does not pin the patch number on this dependency.

All contributions were made by [Kijewski](https://github.com/Kijewski) and included:
- adding a new feature `alloc` which entirely encompasses the existing `std` feature and allows for the crate to use `no_std`; this change is backwards-compatible as it keeps the `std` feature the same
- removal of the `cfg-if` dependency
- replacement of the `pin-utils` dependency with `pin-project-lite` since they achieve the same function in scoped-futures of projecting a future and pin-project-lite is actively maintained

Let me know here once you're able to review these changes and I'll release the new patch. Thanks!

**Comment by weiznich** [comment]
Thanks for the headsup. I had a look at these changes and found they look good, so :+1: from me.

Also thanks for keeping that crate up to date :heart:
